bee-pow/Cargo.toml

```diff
@@ -14,7 +14,8 @@ homepage = "https://www.iota.org"
 bee-crypto = { version = "0.2.1-alpha", path = "../bee-crypto", default-features = false }
 bee-ternary = { version = "0.5.0", path = "../bee-ternary", default-features = false }
 
-iota-crypto = { version = "0.7.0", default-features = false, features = [ "blake2b", "digest" ] }
+# Wait for new release of `iota-crypto` that uses `bee-node:0.5.0`.
```

Comment:
Does this block the PR?

Reply:
Now that @thibault-martinez has bumped the version of `iota-crypto` 💪 everything is fine.